service/gcs/core/gcs/networking.go

```diff
@@ -161,85 +78,30 @@ func (c *gcsCore) generateResolvConfFile(adapter prot.NetworkAdapter) error {
 	}
 	fileContents += fmt.Sprintf("search %s\n", adapter.HostDNSSuffix)
 
-	file, err := c.OS.OpenFile(filepath.Join(baseFilesPath, "etc", "resolv.conf"), os.O_CREATE|os.O_WRONLY, 0700)
+	file, err := c.OS.OpenFile(resolvPath, os.O_CREATE|os.O_WRONLY, 0700)
 	if err != nil {
 		return errors.Wrapf(err, "failed to create resolv.conf file for adapter %s", adapter.AdapterInstanceID)
 	}
 	defer file.Close()
 	if _, err := io.WriteString(file, fileContents); err != nil {
 		return errors.Wrapf(err, "failed to write to resolv.conf file for adapter %s", adapter.AdapterInstanceID)
 	}
+	logrus.Debugf("wrote %s:\n%s", resolvPath, fileContents)
 	return nil
 }
 
-// moveAdapterIntoNamespace moves the given network adapter into the namespace
-// of the container with the given ID.
-func (c *gcsCore) moveAdapterIntoNamespace(container runtime.Container, adapter prot.NetworkAdapter) error {
-	// Get the root namespace, which should be the GCS's current namespace.
-	rootNamespace, err := c.OS.GetCurrentNamespace()
-	if err != nil {
-		return errors.Wrap(err, "failed to get the root namespace")
-	}
-	defer rootNamespace.Close()
-
-	// Get namespace information for the container process.
-	containerNamespace, err := c.OS.GetNamespaceFromPid(container.Pid())
-	if err != nil {
-		return errors.Wrapf(err, "failed to get the namespace of process %d", container.Pid())
-	}
-	defer containerNamespace.Close()
-
-	// Move the interface into the container namespace.
-	link, err := c.getLinkForAdapter(adapter)
-	if err != nil {
-		return errors.Wrapf(err, "failed to get the link for adapter %s in preparation for moving it into the container namespace", adapter.AdapterInstanceID)
-	}
-	addrs, err := link.Addrs(syscall.AF_INET)
-	if err != nil {
-		return errors.Wrapf(err, "failed to get the addrs for adapter %s", adapter.AdapterInstanceID)
-	}
-	gateways, err := link.GatewayRoutes(syscall.AF_INET)
+// instanceIDToName converts from the given instance ID (a GUID generated on
+// the Windows host) to its corresponding interface name (e.g. "eth0").
+func (c *gcsCore) instanceIDToName(id string) (string, error) {
```

Comment:
Golang seems to have a rough rule of most important functions first. Since I was changing this file anyway, I re-orged the order of the functions